spec/vmpooler/api_spec.rb

```diff
@@ -11,13 +11,20 @@ def app()
   describe 'Dashboard' do
 
     context '/' do
+      before { get '/' }
+
+      it { expect(last_response.status).to eq(302) }
+      it { expect(last_response.location).to match(/dashboard/) }
```

Comment:
It's a call to `/`, which should redirect to `dashboard`, so I was attempting to make sure that was set in the `location` headers.

````
$ curl -v --url vmpooler-dev/
````
````
* Hostname was NOT found in DNS cache
*   Trying 10.32.112.37...
* Connected to vmpooler-dev (10.32.112.37) port 80 (#0)
> GET / HTTP/1.1
> User-Agent: curl/7.37.1
> Host: vmpooler-dev
> Accept: */*
>
< HTTP/1.1 302 Found
* Server nginx is not blacklisted
< Server: nginx
< Date: Fri, 10 Apr 2015 17:01:19 GMT
< Content-Type: text/html;charset=utf-8
< Content-Length: 0
< Connection: keep-alive
< Location: http://vmpooler-dev/dashboard/
< X-XSS-Protection: 1; mode=block
< X-Content-Type-Options: nosniff
< X-Frame-Options: SAMEORIGIN
<
* Connection #0 to host vmpooler-dev left intact
````

Is that not sane?